src/api/resources/sampleunitmethods/benthicphotoquadrattransectmethod.py

```diff
@@ -81,6 +81,26 @@ class BenthicPhotoQuadratTransectMethodView(
     serializer_class = BenthicPhotoQuadratTransectMethodSerializer
     http_method_names = ["get", "put", "head", "delete"]
 
+    def edit_sample_unit(self, request, pk):
+        bpqt_record = self.get_queryset().get(pk=pk)
+        image_classification = bpqt_record.image_classification
+        cr = super().edit_sample_unit(request, pk)
+        if image_classification is True:
+            image_classification = True
```

Comment:
What CodeRabbit said

Reply:
I want to force `image_classification` True or False and remove None if it exists.

I will change the line `if image_classification is True:` to `if image_classification`